cucumber/authenticators_jwt/features/step_definitions/authn_jwt_steps.rb

```diff
@@ -26,6 +26,10 @@
   authenticate_jwt_token(jwt_token, service_id)
 end
 
+Given(/I successfully set authn-jwt "([^"]*)" variable in service "([^"]*)" with untrusted TLS server/) do |variable, service_id|
```

Comment:
It's untrusted because we do nothing in order to trust it.
When we will develop proper 3rd party certificates handling the same url will be trusted so I think that we should not "untrusted" world in code. It's just https url that's all.